src/main/scala/intellij/haskell/editor/HaskellCompletionContributor.scala

```diff
@@ -182,6 +184,10 @@ class HaskellCompletionContributor extends CompletionContributor {
                 resultSet.addAllElements(findTopLevelTypeSignatureLookupElements(psiFile).asJavaCollection)
               }
 
+              if (HaskellConsoleView.isConsole(psiFile)) {
```

Comment:
> In the console you do not know which Haskell file is loaded and which imports are manually added. That's what I mean by the context.

If you add the action to load Haskell file in repl again, we know the context. Assuming the user does not load manually other file.